worker/include/RTC/RTCP/CompoundPacket.h

```diff
@@ -12,21 +12,17 @@ namespace RTC { namespace RTCP
 	class CompoundPacket
 	{
 	public:
-		~CompoundPacket();
 
-		uint8_t* GetRaw();
 		size_t GetSize();
 		size_t GetSenderReportCount();
 		size_t GetReceiverReportCount();
 		void Dump();
 		void AddSenderReport(SenderReport* report);
 		void AddReceiverReport(ReceiverReport* report);
 		void AddSdesChunk(SdesChunk* chunk);
-		void Serialize();
+		void Serialize(uint8_t* data);
 
 	private:
-		uint8_t* raw = nullptr;
-		size_t size = 0;
```

Comment:
The reason is because a `CompoundPacket` is created to be immediately sent. There is no need to locally store the serialised data since an external buffer is used for that.